src/pages/outcome/[rowKey].tsx

```diff
@@ -0,0 +1,87 @@
+import { useGameInfoContext } from 'components/Contexts';
+import { OutcomePanel } from 'components/Outcomes';
+import { Layout } from 'components/ui';
+import { MediaService } from 'lib/MediaService';
+import { IAnswer, IImage } from 'models';
+import { GetStaticProps } from 'next';
+import { FC, useEffect } from 'react';
+import { AzureTableService } from 'services/AzureTable/AzureTableService';
+
+interface OutcomeHashPageProps {
+  answers: IAnswer[];
+  persona: string;
+  avatar: IImage | undefined;
+  theme: string;
+}
+
+// Don't pre-render pages at build time.
+export const getStaticPaths = async () => {
+  return {
+    paths: [],
+    fallback: true,
+  };
+};
+
+// Generate Pages when requested
+export const getStaticProps: GetStaticProps = async (context) => {
+  const rowKey = context.params?.rowKey as string;
+
+  if (rowKey) {
+    const azureTableService = new AzureTableService();
+
+    const payload = await azureTableService.getByRowKey(rowKey);
```

Comment:
I do think we could send them to a more friendlier error, this combined with the other issue where you go through the prompts and if it can't generate the outcome page for some reason, we could have this state and the other state lead you to a page where it says "We can't create an outcome for your scenario" or something to that effect.  We could then include a button to start over or something like that.

We could then add that tracking, but I'm not sure what would lead to a scenario where someone would enter a wrong guid.  Unless someone is purely trying to hack the system.  It's not going to be easy to guess the urls, that might be more common if the urls had friendlier names.

Thoughts? We can discuss this further tomorrow if needed, unless you agree with the above approach for this and the other item and then I can get to work on that today.

Reply:
I like idea of an error page. We can have it be semi-generic right now but it's where we send people if we cannot generate the page or cannot find the page.